src/controller/org.controller/org.controller.js

```diff
@@ -50,6 +51,52 @@ async function getCna (req, res) {
     })
 }
 
+// Get details on ID quota for a org with the specified org shortname
+async function getOrgIdQuota (req, res) {
+  const shortName = req.params.shortname
+
+  Org.findOne()
+    .byShortName(shortName)
+    .exec((err, result) => {
+      if (err) {
+        logger.error(err.stack)
+        return res.status(500).send(CONSTANTS.SERVER_RESPONSES.Internal_Server_Error)
+      }
+
+      if (!result) {
+        logger.info(shortName + ' CNA does not exist.')
+        return res.status(404).json({ message: shortName + ' CNA does not exist.' })
+      }
+
+      const returnPayload = { id_quota: undefined, total_reserved: undefined, available: undefined}
+      returnPayload.id_quota = result.policies.id_quota
```

Comment:
Change to 




`const returnPayload = {
        id_quota: result.policies.id_quota,
        total_reserved: undefined,
        available: undefined
      }`